### PR TITLE
pytorch mlp -> main

### LINKED ISSUE #2: PyTorch - MLP

### PR DESCRIPTION
**Pull request by adke**
This is a work in progress model. I still need to train and test the model for proper accuracy. So I am still trying to figure that out. Therefore, I am aware I am still missing key components to the model.

### CODE REVIEW

**Review by mushi251 — COMMENTED** [review]
Line 72: I could be wrong, but I believe that self.fc1 is hidden layer#1 and not an input layer. ff so ,you may have to remove self.fc2 

Line 75: Not sure if you need a dropout layer
